azure-app-service-vnet-template/Infrastructure/vnet.bicep

```diff
@@ -0,0 +1,38 @@
+param vnet_name string
+param vnet_addr_prefixes array
+param location string
+param tags object
+
+resource vnet 'Microsoft.Network/virtualNetworks@2022-01-01' = {
+  name: vnet_name
+  location: location
+  properties: {
+    addressSpace: {
+      addressPrefixes: vnet_addr_prefixes
+    }
+    subnets: [
+      {
+        name: 'todoappsubnet01'
```

Comment:
rename the net and subnet according to the architecture diagram

Reply:
Resolved